caddyhttp/proxy/upstream_test.go

```diff
@@ -316,6 +317,10 @@ func TestParseBlock(t *testing.T) {
 			if _, ok := headers["X-Forwarded-Proto"]; !ok {
 				t.Errorf("Test %d: Could not find the X-Forwarded-Proto header", i+1)
 			}
+
+			if _, ok := headers["X-Forwarded-For"]; ok {
```

Comment:
Test that we do not set the **X-Forwarded-For** when using the transparent Preset